model-test/src/test/java/hu/blackbelt/judo/meta/jsl/runtime/AnnotationTests.xtend

```diff
@@ -371,7 +372,7 @@ class AnnotationTests {
 
             annotation A2(string s) on service::function;
         '''.parse => [
-            m | m.assertError(JsldslPackage::eINSTANCE.annotationArgument, JslDslValidator.TYPE_MISMATCH)
+            m | m.assertError(JsldslPackage::eINSTANCE.annotationArgument, JslDslValidator.TYPE_MISMATCH,"Non-literal at annotation argument 's'. Only literals are allowed in annotation argument.")
```

Comment:
> Non-literal at annotation argument 's'

`1` is a literal. I think the expected error message is invalid. The error message should state that the type of given argument is different then the expected, String type